src/scripts/main.js

```diff
@@ -1 +1,72 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const resolver1 = (resolve, reject) => {
```

Comment:
It's not resolver because it can be rejected. Better use variable names from task -> `firstPromise ` `secondPromise ` `thirdPromise `